k8s/charts/openebs/templates/deployment-admission-server.yaml

```diff
@@ -12,6 +12,9 @@ metadata:
     openebs.io/version: {{ .Values.release.version }}
 spec:
   replicas: {{ .Values.webhook.replicas }}
+  strategy:
+    type: {{ .Values.webhook.upgradeStrategy }}
+    rollingUpdate: null
```

Comment:
should rollingUpdate be set to null .. if the upgradeStrategy is set to  `Recreate`. Having said that, should we even give this option to select the upgradeStrategy or hardcode it to "Recreate" ?

Reply:
We will hardcode it to `Recreate`